src/cloudConsole.ts

```diff
@@ -150,7 +150,7 @@ export async function openCloudConsole(api: AzureAccount, os: OS, files, outputC
 			} else {
 				for (let file of files) {
 					const data = fsExtra.readFileSync(file, { encoding: 'utf8' }).toString();
-					outputChannel.append('\nUpload playbook to CloudShell: ' + file + ' as $HOME/' + path.basename(file));
+					outputChannel.appendLine('\nUpload playbook to CloudShell: ' + file + ' as $HOME/' + path.basename(file));
```

Comment:
shouldn't we now have "upload *file* to CloudShell" as it may not necessarily be playbook?

Reply:
fixed. those part of code not necessary any more. we're not using azure storage uploading